handler.go

```diff
@@ -75,3 +75,34 @@ func EchoBody (w *Response, r *Request) {
 	}
 }
 
+func TextResponse(w *Response, r *Request) {
+	ApplyStatusToResponse(w, StatusOK)
+	w.Header.Add("Content-Type", "text/plain;charset=utf-8")
+	expectedBody := "text response"
+	w.Body = expectedBody
+}
+
+func HTMLResponse(w *Response, r *Request) {
+	ApplyStatusToResponse(w, StatusOK)
+	w.Header.Add("Content-Type", "text/html;charset=utf-8")
+	expectedBody := "<html><body><p>HTML Response</p></body></html>"
+	w.Body = expectedBody
+}
+
+func JSONResponse(w *Response, r *Request) {
+	ApplyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
+	w.Header.Add("Content-Type", "application/json;charset=utf-8")
+	expectedBody := "{\"key1\":\"value1\",\"key2\":\"value2\"}"
+	w.Body = expectedBody
+}
+
+func XMLResponse(w *Response, r *Request) {
+	ApplyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
+	w.Header.Add("Content-Type", "application/xml;charset=utf-8")
+	expectedBody := "<note><body>XML Response</body></note>"
```

Comment:
is the body supposed to be hard coded here? or is it supposed to read the data coming from the request?

Reply:
The body is what was required by the integration specs, but there is no data sent from the request.